openqa-qam-sle

```diff
@@ -0,0 +1,208 @@
+
+            local id
+            id=$(curl -sk -X GET "$URL" | jq -r '.[0].id | select( . != null)')
+
+            [[ -z "$id" ]] && continue
+
+            local json
+            json=$(curl -sk -X GET "https://openqa.suse.de/api/v1/jobs/${id}")
+
+            local issues
+            issues=$(echo "$json" | jq -r '.job.settings | to_entries | .[] | .key + "=" + (.value | @sh)' | grep -E '_TEST_ISSUES=' | perl -lpe "s/.*=//;s/,/ /g" | tr -d "'")
+
+            for issue in $issues; do
+                # echo "- Issue: $issue"
+                [[ $issue -eq $ID ]] && {
+                    echo "Version: '$version' Update: '$ID'"
+                    echo "Build $build contains $ID"
+                    echo "https://openqa.suse.de/tests/overview?distri=sle&version=${version}&build=${build}&groupid=${groupid}"
+
+                    local running_url="https://openqa.suse.de/api/v1/jobs/overview?distri=sle&version=${version}&build=${build}&groupid=${groupid}&state=scheduled&state=running"
+                    local running
+                    running=$(curl -sk -X GET "$running_url" | jq -r '. | length')
+                    [[ $running -gt 0 ]] && {
+                        _warn "RUNNING / SCHEDULED ($running jobs) Awaiting completion..."
+                        echo ""
+                        continue
+                    }
+
+                    local failed_url="https://openqa.suse.de/api/v1/jobs/overview?distri=sle&version=${version}&build=${build}&groupid=${groupid}&result=failed&result=incomplete&result=timeout_exceeded"
+                    local failed
+                    failed=$(curl -sk -X GET "$failed_url" | jq -r '. | length')
+                    [[ $failed -eq 0 ]] && _ok "PASSED"
+                    [[ $failed -gt 0 ]] && _ko "FAILED ($failed jobs)"
+                    echo ""
+                    break 2
+                }
+            done
+        done
+    done
+    echo "---"
+}
+
+search_build_checks() {
+    _title "Build checks"
+    local ReviewRequestID=$1
+    local index
+    index="https://qam.suse.de/testreports/${ReviewRequestID}/build_checks"
+    local logs
+    logs=$(curl -sLk -X GET "$index" | perl -lne '/a href="(.*?\.log)"/ && print "$1"')
+    if [[ -n $logs ]]; then
+        echo "Build checks url: $index"
+        for log in $logs; do
+            echo "curl -sLk '$index/$log' | grep -B1 -A9 -i -E 'summary'"
+            curl -sLk "$index/$log" | grep -B1 -A9 -i -E 'summary' || true
+            echo ""
+        done
+    fi
+    echo "---"
+}
+
+while getopts :a:d: name; do
+    case $name in
+        a) ReviewRequestID="$OPTARG" ;;
+        d) days="$OPTARG" ;;
+        ?)
+            usage
+            exit 1
+            ;;
+    esac
+done
+
+if [[ -z "$ReviewRequestID" ]]; then
+    usage
+    exit 1
+fi
+
+days="${days:-5}"
+
+# ReviewRequestID=$1
+# days=${2:-5}
```

Comment:
Maybe I wasn't clear but I think all commented-out code should be removed.

Reply:
Removed